stan/math/rev/mat/fun/cholesky_decompose.hpp

```diff
@@ -233,8 +233,6 @@ namespace stan {
      * Internally calls llt rather than using cholesky_decompose in order to
      * use selfadjointView<Lower> optimization.
```

Comment:
Is this doc broken now?  You've removed the `selfadjointView<Lower>` bit of the code.